docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,110 @@
+# Release 427 (25 Sep 2023)
+
+## General
+
+* Add support for IPv4 addresses in comparisons using the `contains` functions. ({issue}`18497`)
+* Make plans with complex aggregations deterministic. ({issue}`19103`)
```

Comment:
No need to document this. Also, it's incorrect. Plans are already deterministic -- this just makes how they are *printed* more predictable.